service/events.go

```diff
@@ -60,5 +61,12 @@ func ForwadableHeaders(c echo.Context) []kafka.Header {
 		}
 	}
 
+	if c.Get("x-rh-sources-org-id") != nil {
```

Comment:
One more thing here, up above on Line 59:

The "new" generated xrhid should include the org_id as well. Maybe we can move that extra else statement the end of this. Basically add a flag to see if the xrhid is there, and if it isn't generate it from the available fields (psk + org_id)

Reply:
All right, done!